extensions/twitch.js

```diff
@@ -0,0 +1,297 @@
+  let text = null;
+  let image = null;
+
+  class Twitch {
+    getInfo() {
+      return {
+        id: "twitch",
+        name: "Twitch",
+        color1: "#772CE8",
+        menuIconURI: "https://tekinical.github.io/twitchURI.png",
+        docsURI:
+          "https://github.com/Tekinical/tekinical.github.io/blob/main/Twitch%20Extension%20Doc.md",
+        blocks: [
+          {
+            blockType: Scratch.BlockType.LABEL,
+            text: "Twitch Chat",
+          },
+          {
+            opcode: "setupchat",
+            text: "setup chat [username] chat fade enabled [fade] show bots [bots] prevent clipping [prevent] width [width] height [height] position [position]",
+            blockType: Scratch.BlockType.COMMAND,
+            blockIconURI: "https://tekinical.github.io/twitchURI.png",
+            arguments: {
+              username: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "Twitch Username",
+              },
+              fade: {
+                type: Scratch.ArgumentType.NUMBER,
+                defaultValue: 30,
+              },
+              bots: {
+                type: Scratch.ArgumentType.BOOLEAN,
+                defaultValue: true,
+              },
+              prevent: {
+                type: Scratch.ArgumentType.BOOLEAN,
+                defaultValue: false,
+              },
+              width: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "100%",
+              },
+              height: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "100%",
+              },
+              position: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "absolute",
+              },
+            },
+          },
+          {
+            opcode: "closechat",
+            text: "close chat",
+            blockType: Scratch.BlockType.COMMAND,
+            blockIconURI: "https://tekinical.github.io/twitchURI.png",
+          },
+          {
+            blockType: Scratch.BlockType.LABEL,
+            text: "Overlays",
+          },
+          {
+            opcode: "text",
+            text: "create text overlay with text [text] with italic [italic] with bold [bold] with size [size] with font [font]",
+            blockType: Scratch.BlockType.COMMAND,
+            blockIconURI: "https://tekinical.github.io/twitchURI.png",
+            arguments: {
+              text: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "hi chat",
+              },
+              italic: {
+                type: Scratch.ArgumentType.BOOLEAN,
+                defaultValue: false,
+              },
+              bold: {
+                type: Scratch.ArgumentType.BOOLEAN,
+                defaultValue: false,
+              },
+              size: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "20px",
```

Comment:
I'd suggested adding "pixels" to the block text and then automatically appending "px" to the value; that way it lines up more with how other Scratch blocks work.

Reply:
Completed!